app/src/main/java/org/oppia/app/help/faq/FAQViewModel.kt

```diff
@@ -21,10 +21,10 @@ class FAQViewModel @Inject constructor(
   private fun getRecyclerViewItemList(): ArrayList<FAQItemViewModel> {
     val faqHeaderViewModel = FAQHeaderViewModel()
     arrayList.add(faqHeaderViewModel)
-
     val questions: Array<String> = activity.resources.getStringArray(R.array.faq_questions)
```

Comment:
faq means its question then why we need faq_question, also faqs would be better if more than one in count.

Reply:
An FAQ is not just a question it is Question + Answer. And therefore faq_question and faq_answer

Reply:
Of-cause so that yo may keep it faq and answer or question and answer..